Task 3-2/SwitchCalculator/src/SwitchCalculator.java

```diff
@@ -0,0 +1,39 @@
+import java.util.Scanner;
+
+public class SwitchCalculator {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Write first number");
+        double firstNumber = sc.nextDouble();
+        System.out.println("Choose operation: +, -, *, /, % ");
+        String operation = sc.next();
+        System.out.println("Write second number");
+        double secondNumber = sc.nextDouble();
+        System.out.print("Result=");
+        switch (operation) {
+            case "+":
+                System.out.println(firstNumber + secondNumber);
+                break;
+            case "-":
+                System.out.println(firstNumber - secondNumber);
+                break;
+            case "*":
+                System.out.println(firstNumber * secondNumber);
+                break;
+            case "/":
+                if (secondNumber != 0) {
+                    System.out.println(firstNumber / secondNumber);
+                    break;
```

Comment:
Please remove

Reply:
Why should I? It works with logic that division by zero is not possible. Where is my error?